draft-ietf-anima-brski-prm.md

```diff
@@ -336,84 +320,84 @@ Solution examples based on existing technology are provided with the focus on ex
 # Architectural Overview {#architecture}
 
 For BRSKI with pledge in responder mode, the base system architecture defined in BRSKI {{RFC8995}} is enhanced to facilitate the new use cases in which the pledge acts as server.
-The pledge-responder-mode allows delegated bootstrapping using a registrar-agent instead of a direct connection between the pledge and the domain registrar.
+The responder mode allows delegated bootstrapping using a registrar-agent instead of a direct connection between the pledge and the domain registrar.
 
 Necessary enhancements to support authenticated self-contained objects for certificate enrollment are kept at a minimum to enable reuse of already defined architecture elements and interactions.
 
 For the authenticated self-contained objects used for the certification request, BRSKI-PRM relies on the defined message wrapping mechanisms of the enrollment protocols stated in {{req-sol}} above.
```

Comment:
Make clear normative statements here. My guess is you refer to POP and POI, but based on what now for this solution described? References?

Reply:
Yes, this is a reference to POI and POP achieved by the authenticated self-contained objects. The wrapping mechanism is defined in this document and refers to the requirements in section 4. I will change the sentence to 

_For the authenticated self-contained objects used for the certification request, BRSKI-PRM pledges MUST used the format defined in section 6.1.3 to address the requirements stated in section 4 above._

Reply:
Looks like this was dropped. Overall, I think anything about "objects produced or consumed" should be removed from Section 5, as you explicitly have Section 6 to contain the definitions for the "Bootstrapping Data Objects". Now we only have redundancy with not giving enough information to understand it Section 5.

Reply:
After rereading it with the comment above, I changed it to 

_The format of the bootstrapping objects produced or consumed by the pledge is based on JOSE {{RFC7515}} and further specified section 6 to address the requirements stated in section 4 above._ 

to make the reader at least aware, that the definition is done in a later section.